### ISSUE TITLE
Version not supported for appPlatform[RabbitiOS]/appVersion

### ISSUE THREAD

**Issue by gmcsnetwork** [mdesilva/AmazonFlexUnlimited#90]
Anyone getting the following errors? It appears the application was recently updated 

errorCode': None, 'message': 'Version not supported for appPlatform[RabbitiOS]/appVersion[2.88.2].

**Comment by gmcsnetwork** [comment]
For anyone else having issues I updated the versions on the following two places and was able to get the script to work again. Someone more familiar with the script, please let me know if I missed anything. 

From 
APP_VERSION = "303338310"
"User-Agent": "iOS/15.2 (iPhone Darwin) Model/iPhone Platform/iPhone13,3 RabbitiOS/2.88.2",

To
APP_VERSION = "303662677"
"User-Agent": "iOS/15.2 (iPhone Darwin) Model/iPhone Platform/iPhone13,3 RabbitiOS/3.88.4",

**Comment by Hrasseee** [comment]
how did you find that out? I was breaking my head here

**Comment by gmcsnetwork** [comment]
The error in the response I was getting indicated an issue with the version of the "app", While troubleshooting I went to AF site, downloaded their Android app APK file and got the new version number and compared it to what was in the script and tried changing. A bit of trial and error....

**Comment by JustinBoshans** [comment]
> For anyone else having issues I updated the versions on the following two places and was able to get the script to work again. Someone more familiar with the script, please let me know if I missed anything.
> 
> From APP_VERSION = "303338310" "User-Agent": "iOS/15.2 (iPhone Darwin) Model/iPhone Platform/iPhone13,3 RabbitiOS/2.88.2",
> 
> To APP_VERSION = "303662677" "User-Agent": "iOS/15.2 (iPhone Darwin) Model/iPhone Platform/iPhone13,3 RabbitiOS/3.88.4",

this simple fix worked for me. Thank you so much!

**Comment by travislazzaro** [comment]
@gmcsnetwork do you know where i could find "FLEX_VERSION: number?

I have another script but im not sure if I need to update it.

APP_NAME = "com.amazon.rabbit"
APP_VERSION = "303338310"
FLEX_VERSION = "3.88.1.41.0"

APP_VERSION is the old number, but I'm wondering what FLEX_VERSION is?

**Comment by gmcsnetwork** [comment]
When I first encounter the version error message I went to the AF site and downloaded the latest app for Android and used the version numbers in the apk file name.

https://logistics.amazon.com/app/download-app


AmazonFlex-3.88.4.34.0-303662677-Standalone-DefaultPerm-Prod-release-signed.apk

**Comment by travislazzaro** [comment]
Oh yes I see now. Thanks. Did you find a workaround for the captcha issue?

**Comment by gmcsnetwork** [comment]
No, just stopping the script when I get the 307 response code, accepting a random offer using the app, completing the captcha, canceling the offer and starting the script again.

**Comment by travislazzaro** [comment]
Change version numbers in FlexUnlimited.py

**Comment by inazeem** [comment]
I have changed the version to 3.88 and it worked fine.
